hazelcast-sql/pom.xml

```diff
@@ -479,7 +497,15 @@
             <artifactId>slf4j-api</artifactId>
             <version>${slf4j.version}</version>
         </dependency>
-
+        <dependency>
+            <groupId>com.jayway.jsonpath</groupId>
+            <artifactId>json-path</artifactId>
```

Comment:
JSON Path isn't standardized. The SQL specification contains also JSON Path specification. Are we confident that this implementation conforms to the SQL standard?

Reply:
Hard to say if they've used SQL TR for this, but its advertised as a direct port of JSONPath library created by the author of JSONPath "standard". Its also the only implementation I've found for Java.

Reply:
For array ranges, SQL/JSONpath uses `$[m to n]`. This library uses `$[m:n]`. It seems that this library also doesn't support predicates, which are very useful, such as `$[*] ? (@.salary > 10)`